### ISSUE TITLE
Relic data applies its effects twice

### ISSUE THREAD

**Issue by LaVeyanFiend** [CleverRaven/Cataclysm-DDA#42452]
# Describe the bug

Items with relic data apply their effects twice.

# Steps To Reproduce

1. Spawn in a ring of [stat] +4
2. Wear it.
3. Your stat will increase by 8.

# Expected behavior

Relic data applies only once.

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1909
- Game Version: 0.E-4401-gea9b62d [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Magiclysm [magiclysm]
]

# Additional context

**Comment by UmbralReaper** [comment]
Similar to #42390

**Comment by anothersimulacrum** [comment]
Duplicate of #42390
